src/styles/blocks/main-block.scss

```
@@ -0,0 +1,47 @@
+* {
+  box-sizing: border-box;
+}
+
+.content {
+  @extend %align-ver-hor;
+  @extend %text-number;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 $padding-sides;
+  gap: $gap-between-cell;
+  width: $widthCalendar;
+  @for $index from 1 through $all-days {
+    & :nth-child(#{$index}) {
+      @extend %align-ver-hor;
+      @include cell-square($size-cell);
+      background-color: $color-cell;
+      border: $border-cell;
+      cursor: pointer;
+      transition-duration: 0.5s;
+      &::before {
```

Always add styles that you apply transition for:

```suggestion
      transition: transform 0.5s, background-color 0.5s;
      
      &::before {
```